burger.css

```diff
@@ -0,0 +1,58 @@
+   
+body {
+  background: #E5E5E5;
```

Comment:
Codacy found an issue: [Expected "#E5E5E5" to be "#e5e5e5" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)